pkg/transformer/openshift/openshift_test.go

```diff
@@ -44,6 +46,29 @@ func newServiceConfig() kobject.ServiceConfig {
 		Privileged:    true,
 		Restart:       "always",
 		User:          "user", // not supported
+		Stdin:         true,
+		Tty:           true,
+	}
+}
+
+func TestOpenShiftUpdateKubernetesObjects(t *testing.T) {
+	var object []runtime.Object
+	o := OpenShift{}
+	serviceConfig := newServiceConfig()
```

Comment:
Fixed for `TestOpenShiftUpdateKubernetesObjects`, log messages for other tests should go in another PR. Tracked at #351